ccmlib/node.py

```diff
@@ -580,6 +580,17 @@ def stop(self, wait=True, wait_other_notice=False, gently=True):
         else:
             return False
 
+    def wait_for_compactions(self):
+        """
+        Wait for all compactions to finish on this node.
+        """
+        pattern = re.compile("pending tasks: 0")
+        while True:
+            output, err = self.nodetool("compactionstats", capture_output=True)
+            if pattern.search(output):
+                break
+            time.sleep(30)
```

Comment:
Can you bump this down to 10 or 15 seconds? And can you add a cluster.wait_for_compactions that just calls wait_for_compactions on each node?